### PR TITLE
fix: remove default spec of kubevirt chart

### PR DESCRIPTION
**Pull request by guangbochen**
https://github.com/rancher/harvester/pulls
1. set the default kubevirt spec to empty as this is more preferred (in this way, bump kubevirt will not introduce unexpected bugs and the user can always set those values via custom config)
1. update the chart Q&A with changing the first option to enable or disable each component

### CODE REVIEW

File: deploy/charts/harvester/charts/kubevirt/values.yaml [hunk 1]
```diff
@@ -16,231 +16,231 @@ fullnameOverride: ""
...
+#    ##
+#    # cpuRequest: "100m"
+#
+#    ## Specify the machine type regex for validating the emulation of VirtualMachineInstance,
+#    ## defaults to discover automatically, "pseries*" if arch is ppc64le, otherwise "q35*,pc-q35*".
+#    ##
+#    emulatedMachines: []
+#
+#    ## Specify the machine type for filling VirtualMachineInstance(`spec.domain.machine.type`),
+#    ## defaults to discover automatically, "pseries" if arch is ppc64le, otherwise "q35".
```
Comment (gitlawr) [inline deploy/charts/harvester/charts/kubevirt/values.yaml]: Comment out in [this format](https://github.com/rancher/harvester/blob/cc3244910710325253abf19c8f4f813af3ec90ec/deploy/charts/harvester/charts/kubevirt/values.yaml#L19-L22) for a better look

Comment (guangbochen) [reply deploy/charts/harvester/charts/kubevirt/values.yaml]: updated, thanks

**Review by gitlawr — APPROVED** [review]
LGTM

**Review by thxCode — APPROVED** [review]
LGTM, What I want to point out is keeping a default/empty KubeVirt CR means that the default values are controlled by kubevirt-operator, therefore, when upgrading kubevirt-operator, it may bring uncertain default value changes.

**Pull request merged**